src/geo/leaflet/leaflet-tiled-layer-view.js

```diff
@@ -25,7 +25,7 @@ LeafletTiledLayerView.prototype = _.extend(
   LeafletLayerView.prototype,
   {
     _createLeafletLayer: function () {
-      return new L.TileLayer(_getUrlTemplate(this._isHdpi(), this.model), _generateLeafletLayerOptions.call(this, this.model));
+      return new L.TileLayer(_getUrlTemplate.call(this), _generateLeafletLayerOptions.call(this));
```

Comment:
This a way to declare private functions without having them on the object's `prototype`. I do agree with the fact that therefore we can not consider it a "pure" function. However, as we have talked, I think that if we do not use this approach anywhere else in cartodb.js, it would be better not to use it in this what.

Reply:
And about what you mentioned about how to declare functions. I do agree that functions must have a name so they are easy to debug, I changed that. However, I prefer to assign the function to a variable because then, they are not hoisted, and they can not be used before they have been declared, and I do like this approach.